pkg/kubectl/cmd/rollout/rollout_pause.go

```diff
@@ -58,14 +58,22 @@ kubectl rollout pause deployment/nginx`
 func NewCmdRolloutPause(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	opts := &PauseConfig{}
 
+	allErrs := []error{}
```

Comment:
I was talking about `cmd.Run`, not `Run<CmdName>`. Just before you call `CompletePause`, define `allErrs`. Won't be a big trouble here probably but in general it's a very good practice to have the scope of each variable reduced to only where it's necessary. Leads to more maintainable code.